actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/OrgBulkUploadBackGroundJobActor.java

```diff
@@ -67,22 +72,60 @@ private void processOrg(BulkUploadProcessTask task) {
     String data = task.getData();
     try {
       Map<String, Object> orgMap = mapper.readValue(data, Map.class);
+      Map<String, Object> csvMap =
```

Comment:
Rework not as expected.

Object getSystemSettingByFieldKey(getActorRef(ActorOperations.GET_SYSTEM_SETTING.getValue()), "orgProfileConfig", "csv.mandatoryColumns", String[].class)